<dup> Update module github.com/golang/mock to v1.6.0

adarshsrinivasan opened the pull request:
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [github.com/golang/mock](https://togithub.com/golang/mock) | require | minor | `v1.5.0` -> `v1.6.0` |

---

### Release Notes

<details>
<summary>golang/mock</summary>

### [`v1.6.0`](https://togithub.com/golang/mock/releases/v1.6.0)

[Compare Source](https://togithub.com/golang/mock/compare/v1.5.0...v1.6.0)

#### Changelog

[`317c030`](https://togithub.com/golang/mock/commit/317c030) Best effort guesses for output package path ([#&#8203;547](https://togithub.com/golang/mock/issues/547))
[`c59ba11`](https://togithub.com/golang/mock/commit/c59ba11) add ARM to support apple M1 chip to releaser ([#&#8203;562](https://togithub.com/golang/mock/issues/562))
[`58935d8`](https://togithub.com/golang/mock/commit/58935d8) add a basic CONTRIBUTING.md ([#&#8203;535](https://togithub.com/golang/mock/issues/535))
[`a5582fc`](https://togithub.com/golang/mock/commit/a5582fc) add docs on 1.16 install and adding to PATH ([#&#8203;534](https://togithub.com/golang/mock/issues/534))
[`0cd3aaf`](https://togithub.com/golang/mock/commit/0cd3aaf) add flags documentation ([#&#8203;539](https://togithub.com/golang/mock/issues/539))
[`64b0b80`](https://togithub.com/golang/mock/commit/64b0b80) add notes and error helper for vendor+reflect error ([#&#8203;567](https://togithub.com/golang/mock/issues/567))
[`e303461`](https://togithub.com/golang/mock/commit/e303461) add type information to error messages ([#&#8203;559](https://togithub.com/golang/mock/issues/559))
[`0cdccf5`](https://togithub.com/golang/mock/commit/0cdccf5) feat add InAnyOrder matcher ([#&#8203;546](https://togithub.com/golang/mock/issues/546))
[`82ce4a7`](https://togithub.com/golang/mock/commit/82ce4a7) feat validate Do & DoReturn args ([#&#8203;558](https://togithub.com/golang/mock/issues/558))
[`93308c3`](https://togithub.com/golang/mock/commit/93308c3) fix broken badge ([#&#8203;525](https://togithub.com/golang/mock/issues/525))
[`9336b7e`](https://togithub.com/golang/mock/commit/9336b7e) fix error message in parse.go ([#&#8203;540](https://togithub.com/golang/mock/issues/540))
[`ab03293`](https://togithub.com/golang/mock/commit/ab03293) fix ill-formatted message with fmt-verbs like %s ([#&#8203;564](https://togithub.com/golang/mock/issues/564))
[`bb5fd5e`](https://togithub.com/golang/mock/commit/bb5fd5e) fix linter errors ([#&#8203;552](https://togithub.com/golang/mock/issues/552))
[`aba2ff9`](https://togithub.com/golang/mock/commit/aba2ff9) fix parse array with the external const correctly ([#&#8203;569](https://togithub.com/golang/mock/issues/569))
[`6ff1070`](https://togithub.com/golang/mock/commit/6ff1070) fix parse arrays with const length correctly ([#&#8203;520](https://togithub.com/golang/mock/issues/520))
[`7f5f64d`](https://togithub.com/golang/mock/commit/7f5f64d) fixup some docs and templates ([#&#8203;524](https://togithub.com/golang/mock/issues/524))
[`7078515`](https://togithub.com/golang/mock/commit/7078515) refactor go:generate lines so they are consistently placed ([#&#8203;527](https://togithub.com/golang/mock/issues/527))
[`7105dde`](https://togithub.com/golang/mock/commit/7105dde) refactor mockgen and cleanup ([#&#8203;536](https://togithub.com/golang/mock/issues/536))
[`f36d14a`](https://togithub.com/golang/mock/commit/f36d14a) test(sample/user_test.go): minor correction at t.Errorf ([#&#8203;544](https://togithub.com/golang/mock/issues/544))
[`ef4ad87`](https://togithub.com/golang/mock/commit/ef4ad87) update CI for 1.16 ([#&#8203;526](https://togithub.com/golang/mock/issues/526))
[`ad820b0`](https://togithub.com/golang/mock/commit/ad820b0) update Finish docs for Go1.14+ ([#&#8203;556](https://togithub.com/golang/mock/issues/556))
[`2421472`](https://togithub.com/golang/mock/commit/2421472) update dependencies ([#&#8203;528](https://togithub.com/golang/mock/issues/528))
[`953a5bb`](https://togithub.com/golang/mock/commit/953a5bb) update user mock to be in test package ([#&#8203;566](https://togithub.com/golang/mock/issues/566))
[`d19a212`](https://togithub.com/golang/mock/commit/d19a212) upgrade dependencies ([#&#8203;557](https://togithub.com/golang/mock/issues/557))

</details>

---

### Configuration

📅 **Schedule**: At any time (no schedule defined).

🚦 **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, click this checkbox.

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

adarsh-zededa commented:
### ⚠ Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: go.sum

```
Command failed: go get -d ./...
Could not forward go, probably wrong PATH variable.

```

Commit:
Update module github.com/golang/mock to v1.6.0

| datasource | package                | from   | to     |
| ---------- | ---------------------- | ------ | ------ |
| go         | github.com/golang/mock | v1.5.0 | v1.6.0 |
